spec/index.html

```diff
@@ -600,7 +600,13 @@ <h3>IRIs</h3>
         <dfn data-lt="URI" data-lt-noDefault><abbr title="Uniform Resource Identifier">URI</abbr>s</dfn>
         [[RFC3986]] that permits a wider range of Unicode characters.
         Every absolute URI and URL is an IRI, but not every IRI is an URI.
-        In RDF, IRIs are used as <em>IRI references</em>, as defined in [[RFC3987]].
+        In RDF, IRIs are used as <dfn data-lt="iri reference">IRI references</dfn>, as defined in [[RFC3987]]
+        <a data-cite="RFC3987#section-1.3">section 1.3</a>.
+        An IRI reference is common usage of an Internationalized Resource Identifier.
+        An IRI reference may be absolute or relative.
+        However, the <a>IRI</a> that results from such a reference
+        only includes absolute IRIs; any <a>relative IRI references</a> are
+        resolved to their absolute form.
```

Comment:
Perhaps if we simplify it to the following:

```suggestion
        An IRI reference refers to either <a>IRI</a> or <a>relative IRI reference</a>,
        as described by the <em>IRI-reference</em> production in <a href="#iri-abnf" class="sectionRef"></a>.
        The abstract syntax uses only fully resolved <a>IRIs</a>.
```

Reply:
```suggestion
        An IRI reference refers to either an absolute <a>IRI</a> or a <a>relative IRI reference</a>,
        as described by the <em>IRI-reference</em> production in <a href="#iri-abnf" class="sectionRef"></a>.
        The abstract syntax uses only fully resolved, absolute <a>IRIs</a>.
```